sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -344,6 +344,8 @@ object FunctionRegistry {
     expression[Length]("length"),
     expression[Levenshtein]("levenshtein"),
     expression[Like]("like"),
+    expression[LikeAll]("like_all"),
+    expression[NotLikeAll]("not_like_all"),
```

Comment:
Do we need to register these UDFs to the `FunctionRegistry`?

Reply:
Maybe, but I want to hear your opinions. @maropu @wangyum @dongjoon-hyun @cloud-fan

Reply:
Make sense. I will not register like_all as function.